doc/libpmempool.3.md

```diff
@@ -514,6 +571,29 @@ utilized for storing internal metadata of the pool part files.
 The function returns either 0 on success or -1 in case of error
 with proper *errno* set accordingly.
 
+The !pmempool_transform_progress function is a variant of !pmempool_transform
```

Comment:
I think you could somehow merge those two sections together so that we don't have copy/pasted text.